content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -26,7 +26,7 @@ Your cluster must be running the CoreDNS add-on.
 ## Introduction
 
 DNS is a built-in Kubernetes service launched automatically
-using the _addon manager_ [cluster add-on](http://releases.k8s.io/master/cluster/addons/README.md).
+using the _addon manager_ [cluster add-on](https://releases.k8s.io/master/cluster/addons/README.md).
```

Comment:
So, normally, I'd suggest this more correct link:
```markdown
using the _addon manager_ [cluster add-on](https://releases.k8s.io/master/cluster/addons/addon-manager/README.md)
```

but **there's a problem**.

> DNS is a built-in Kubernetes service launched automatically using the _addon manager_

usually isn't true.

I think the whole page needs more work. We don't yet have a concept page about cluster DNS; if we had one, the page introduction could link to it.

For now, I suggest dropping this paragraph and making a new summary of why people run CoreDNS. You could mention that `kubeadm` automatically deploys this add-on.

Reply:
is it better to change it to a direct link?
https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/addon-manager/README.md